build/build.sh

```diff
@@ -25,4 +25,7 @@ echo "weave:x:$uid:$gid::$WEAVE_SRC:/bin/sh" >>/etc/passwd
 echo "weave:*:::::::" >>/etc/shadow
 echo "weave	ALL=(ALL)	NOPASSWD: ALL" >>/etc/sudoers
 
+# Allow full write access to the Go folders for the user Weave
+chmod -R 777 /usr/local/go
```

Comment:
Yes, I know that it's full control for everyone, I guess that was just a quick fix early on to get it working.

Now the user that's running go (weave) has to be able to write to the `/usr/local/go/pkg/linux_${ARCH}_cgo` packages IIRC and by default only root can do that. amd64 has this precompiled